eslint.config.mjs

```diff
@@ -0,0 +1,30 @@
+import globals from "globals";
+import { ember } from "ember-eslint";
```

Comment:
this library tracks / supports all the currently active blueprints -- it's way easier to adopt in projects migrating because I don't need to add/upgrade/delete the 6+ dependencies that are required to use eslint in the default blueprint